content/en/profiler/connect_traces_and_profiles.md

```diff
@@ -66,14 +66,14 @@ To enable the new [timeline feature](#span-execution-timeline-view) (beta):
 {{< /programming-lang >}}
 {{< programming-lang lang="nodejs" >}}
 
-Code Hotspots (beta) identification is NOT enabled by default when you [turn on profiling for your Node.js service][1]. Enable it by setting this additional environment variable:
+Code Hotspots identification is enabled by default when you [turn on profiling for your Node.js service][1] on Linux and macOS. The feature is not available on Windows.
+
+Requires `dd-trace-js` version 5.0.0+, 4.24.0+ or 3.45.0+. It also works with versions 4.17.0+ or 3.38.0+, but with them you need to enable it by setting this additional environment variable:
```

Comment:
```suggestion
Requires `dd-trace-js` version 5.0.0+, 4.24.0+ or 3.45.0+.
```

Reply:
We try to not document old beta requirements because it creates noise and is not a best practice (if you are on a supported beta version, then you should upgrade to GA instead of turning on an env var)